tests/cnf/core/network/metallb/tests/frrk8-tests.go

```diff
@@ -372,10 +385,20 @@ var _ = Describe("FRR", Ordered, Label(tsparams.LabelFRRTestCases), ContinueOnFa
 					// Get the routes
 					frrNodeState, err := metallb.ListFrrNodeState(APIClient)
 					Expect(err).ToNot(HaveOccurred(), "Failed to verify BGP routes")
-					fmt.Println("frrNodeState[0].Objects.Status.RunningConfig",
-						frrNodeState[0].Object.Status.RunningConfig)
 
-					return frrNodeState[0].Object.Status.RunningConfig
+					var frrWorker1Status *metallb.FrrNodeStateBuilder
```

Comment:
IMO better to create help func to move this logic, something like:
```
func findNodeByName(frrNodeState []metallb.FrrNodeStateBuilder, nodeName string) *metallb.FrrNodeStateBuilder {
    for _, node := range frrNodeState {
        if node.Definition.Name == nodeName {
            return node
        }
    }
    return nil
}
```
and then it should be something like:
```
 frrWorker1Status := findNodeByName(frrNodeState, cnfWorkerNodeList[1].Definition.Name)
    Expect(frrWorker1Status).ToNot(BeNil(), fmt.Sprintf("Node with name %s not found", cnfWorkerNodeList[1].Definition.Name))
```

Reply:
Updated